### PR TITLE
manifest: updated openthread module

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
Regular upmerge of openthread module.

Signed-off-by: Kamil Kasperczyk <kamil.kasperczyk@nordicsemi.no>

### CODE REVIEW

**Comment by kkasperczyk-no** [comment]
Associated OpenThread PR: https://github.com/zephyrproject-rtos/openthread/pull/38

**Comment by kkasperczyk-no** [comment]
@rlubos @edmont FYI

**Review by rlubos — APPROVED** [review]
LGTM, seems to work fine.

**Review by edmont — APPROVED** [review]

**Comment by kkasperczyk-no** [comment]
CI is green

File: west.yml [hunk 1]
```diff
@@ -109,7 +109,7 @@ manifest:
       revision: 2cee5f7295ff0ff804bf06fea5de006bc7cd121e
       path: modules/lib/loramac-node
     - name: openthread
-      revision: d44ee763b7e8fea29b54dd064342c771161afcab
+      revision: pull/38/head
```
Comment (rlubos) [inline west.yml:112]: Please update SHA: `fd27fc3a7758d409e00d0e8d43f3f8364ce6c7e9`

**Review by rlubos — APPROVED** [review]

**Review by jukkar — APPROVED** [review]

**Comment by rlubos** [comment]
@jukkar Could we get this in?

**Comment by rlubos** [comment]
@jukkar Thanks!

**Pull request merged**